packages/toggle/src/Toggle.tsx

```diff
@@ -61,63 +52,41 @@ type ToggleProps = {
  */
 
 const Toggle = (props: ToggleProps) => {
-  const {
-    'aria-label': ariaLabel,
-    'aria-labelledby': ariaLabelledby,
-    checked,
-    children,
-    className,
-    disabled,
-    id = 'id',
-    onChange,
-    toggleOffText = 'Off',
-    toggleOnText = 'On',
-    'data-test-id': testId = 'toggle',
-  } = props;
+  const { className, isDisabled = false, autoFocus, 'data-test-id': testId = 'toggle' } = props;
+
+  const id = useId();
   const state = useToggleState(props);
   const inputRef = useRef<HTMLInputElement>(null);
-  const { inputProps } = useSwitch(props, state, inputRef);
+  const { inputProps } = useSwitch({ ...props, id }, state, inputRef);
+
   const classes = cx(
     styles.Toggle,
     className,
-    checked && styles['Toggle--on'],
-    disabled && styles['Toggle--disabled']
+    state.isSelected && styles['Toggle--on'],
+    isDisabled && styles['Toggle--disabled']
```

Comment:
i know i ran into this with the select components, that react-aria wants to use `isDisabled` under the hood instead of `disabled`. besides that `Restrict` component in gonfalon, i'm not sure if there are other scenarios where we'd run into a mismatch of expectations between `disabled`/`isDisabled`. i wonder if it's worth us always exposing `disabled` and mapping to `isDisabled` under the hood for react-aria, or if we should use `isDisabled` for launchpad components until the need arises to switch to `disabled`?

Reply:
I believe it makes more sense to keep it as `isDisabled` and handle mapping in consuming apps if needed. Under the hood, the property is doing more business logic work the native prop `disabled` might imply. Also, other boolean "flag" props use the `is` prefix internally. Not everywhere, but I'd like to continue moving toward that pattern where it makes sense.

Reply:
that's reasonable. if using `isDisabled` is the preference, then in the near future we probably want to update `Restrict` to handle that; i had avoided doing that so as not to impact a bunch of components in gonfalon, but maybe it's the right move now.